atlite/datasets/era5.py

```diff
@@ -93,7 +93,7 @@ def prepare_meta_era5(xs, ys, year, month, module):
         ds = _add_height(ds)
 
         t = pd.Timestamp(year=year, month=month, day=1)
-        ds['time'] = pd.date_range(t, pd.offsets.shift_month(t, months=1),
+        ds['time'] = pd.date_range(t, t + pd.DateOffset(months=1),
```

Comment:
At least on pandas 0.23.4 this does not work:
```python
In [4]: pd.Timestamp("2019-10-01") + pd.offsets.DateOffset(month=1)                            
Out[4]: Timestamp('2019-01-01 00:00:00')
```
Instead of incrementing by a month, it seems to set the month! Went done that rabbit-hole myself.